services/viva/microservice/src/lambdas/syncWorkflow.ts

```diff
@@ -84,7 +55,7 @@ function getSubmittedOrProcessingOrOngoingCases(personalNumber) {
   return dynamoDb.call('query', queryParams);
 }
 
-function updateCaseDetailsWorkflow(caseKeys, newWorkflow) {
+function updateCaseDetailsWorkflow(caseKeys: CaseKeys, newWorkflow: any) {
```

Comment:
caseKeys kan bara vara keys då du redan typar det fint

any is not legal :)

workflow finns typat se types/viva** så hittar du det säkert